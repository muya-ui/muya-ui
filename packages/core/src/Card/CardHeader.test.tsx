import React from 'react';
import renderer from 'react-test-renderer';
import CardHeader from './CardHeader';

describe('CardHeader', () => {
  it('should render success', function() {
    const wrapper = renderer
      .create(
        <>
          <CardHeader>测试</CardHeader>
          <CardHeader
            src="//qhyxpicoss.kujiale.com/2019/03/26/LSMM56IKAQBZMRC3AAAAADQ8_3840x2160.jpg@!212"
            height={100}
          >
            测试
          </CardHeader>
          <CardHeader
            src="//qhyxpicoss.kujiale.com/2019/03/26/LSMM56IKAQBZMRC3AAAAADQ8_3840x2160.jpg@!212"
            height="d"
          >
            测试
          </CardHeader>
        </>,
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
