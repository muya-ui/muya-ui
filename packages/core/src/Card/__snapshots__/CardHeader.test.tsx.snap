// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader should render success 1`] = `
Array [
  .c0 {
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  -webkit-transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
  transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
}

<div
    className="c0 muya-img-loading"
  >
    测试
  </div>,
  .c0 {
  height: 100px;
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  -webkit-transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
  transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
}

<div
    className="c0 muya-img-loading"
  >
    测试
  </div>,
  .c0 {
  height: d;
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  -webkit-transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
  transition: background-image 300ms cubic-bezier(0.0,0,0.2,1);
}

<div
    className="c0 muya-img-loading"
  >
    测试
  </div>,
]
`;
